packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -190,12 +190,18 @@ export class DocumentDeltaConnection
         return this.details.serviceConfiguration;
     }
 
+    private checkNotClosed() {
+        assert(!this.disposed, "not disposed");
```

Comment:
Can you please change the assert message to "Connection disposed"?

Reply:
sure